### PR TITLE
lite: use flags instead of parsing args

### PR DESCRIPTION
**Pull request by melekes**
also, remove init-force and update-force: seem redundant to me. feel
free to revert.

see my notes on UpdateLiteDBToLatestHeader and usage of --url and
--force flags, which are not currently used at all.

### CODE REVIEW

**Comment by melekes** [comment]
```
⚡ » ../build/relayer lite init ibc0 --height=15 --hash=EB41A099DB213D2AF04931D5227B83C3BEBF97AB05C0BBC373E536F7E4DFE0DD --home=$RLY
I[2020-02-12|11:26:57.469] VerifyHeader                                 height=85417CA9B05169402CC4898E289820121CB46973D59A7C26E0DE5AB1E3247B2E newVals=1D1FE976EC97D7472FA043A9AAA940FB36665D715BD083C0525FA1068D8553C1
D[2020-02-12|11:26:57.471] Verify newHeader against lastHeader          lastHeight=15 lastHash=EB41A099DB213D2AF04931D5227B83C3BEBF97AB05C0BBC373E536F7E4DFE0DD newHeight=30 newHash=85417CA9B05169402CC4898E289820121CB46973D59A7C26E0DE5AB1E3247B2E
I[2020-02-12|11:26:57.480] Advanced to new state                        height=30 hash=85417CA9B05169402CC4898E289820121CB46973D59A7C26E0DE5AB1E3247B2E
```

File: cmd/lite.go [hunk 1]
```diff
@@ -31,10 +31,23 @@ import (
 )
 
 var (
-	flagHeight = "height"
-	flagHash   = "hash"
-	flagURL    = "url"
-	flagForce  = "force"
+	flagHeight          = "height"
+	heightDesc          = "Trusted header's height"
```
Comment (melekes) [inline cmd/lite.go]: don't think you need constants for all the fields unless they are used repeatedly

File: cmd/lite.go [hunk 2]
```diff
@@ -73,50 +79,77 @@ func initLiteCmd() *cobra.Command {
 
 			db, df, err := chain.NewLiteDB()
 			if err != nil {
-				return fmt.Errorf("can't open db connection: %w", err)
```
Comment (melekes) [inline cmd/lite.go]: errors should be as descriptive as possible. without wrapping user will get an error like : "resource is temporary unavailable" which does not tell you what resource exactly!

File: cmd/lite.go [hunk 3]
```diff
@@ -73,50 +79,77 @@ func initLiteCmd() *cobra.Command {
...
+				}
+			default: // return error
+				return errInitWrongFlags
 			}
 
 			return nil
 		},
 	}
 
+	cmd.Flags().Int64(flagHeight, heightDefault, heightDesc)
```
Comment (melekes) [inline cmd/lite.go]: what's wrong with using init?

File: cmd/lite.go (hunk 1 above)
Comment (jackzampolin) [reply cmd/lite.go]: We are using them a couple of times, not necessarily needed but figured I'd throw them up top. Happy to change here.

File: cmd/lite.go (hunk 2 above)
Comment (jackzampolin) [reply cmd/lite.go]: So I've wrapped the error here at the `chain.NewLiteDB` level which should handle this case.

File: cmd/lite.go (hunk 3 above)
Comment (jackzampolin) [reply cmd/lite.go]: I find that keeping the flags defined on the commands where they are defined makes it more clear as to which flags are where as opposed doing it all in init. Picked this style up from the SDK and I kinda like it.

**Pull request merged**